src/vivarium_census_prl_synth_pop/model_specifications/model_spec.yaml

```diff
@@ -37,4 +37,4 @@ configuration:
             day: 1
         step_size: 28 # Days
     population:
-        population_size: 2_000
+        population_size: 100_000
```

Comment:
Is this something we want to change?  I don't have a problem with it personally because we seem to be finding more errors when we increase population size as it allows us to catch more edge cases.

Reply:
Uhh, not intentionally - I just forgot to change it back